packages/fbjs/src/__forks__/setImmediate.js

```diff
@@ -14,4 +14,6 @@
 // setimmediate adds setImmediate to the global. We want to make sure we export
 // the actual function.
 require('setimmediate')
-module.exports = global.setImmediate;
+module.exports = function(fn, ms) {
+  global.setImmediate(fn, ms);
+};
```

Comment:
oh, always forgetting about possibility to pass more args, never used it, fixed!